src/switch_rtp.c

```diff
@@ -9416,30 +9417,13 @@ static int rtp_common_write(switch_rtp_t *rtp_session,
 		//	//switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_WARNING, "SEND %u\n", ntohs(send_msg->header.seq));
 		//}
 
-		if (rtp_session->rtp_bugs & RTP_BUG_SEND_NORMALISED_TIMESTAMPS) {
-			{
-				uint32_t ts = normalised_ts_get_next(rtp_session);
-#if DEBUG_RTP
-				switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(rtp_session->session), SWITCH_LOG_NOTICE, "RTP: NORM ts: %u %p/%p\n", ts, (void*)rtp_session->session, (void*)rtp_session); 
-#endif
-				send_msg->header.ts = htonl(ts);
-				if (switch_socket_sendto(rtp_session->sock_output, rtp_session->remote_addr, 0, (void *) send_msg, &bytes) != SWITCH_STATUS_SUCCESS) {
-					rtp_session->seq -= delta;
-					ret = -1;
-					goto end;
-				}
-				normalised_ts_commit(rtp_session, ts);
-			}
-#endif
```

Comment:
This enclose the RTP_WRITE_PLOSS macro condition in line 9397. You need to properly close this macro condition

Reply:
Thanks.